simtools/data_model/validate_data.py

```diff
@@ -141,26 +183,13 @@ def _validate_data_dict(self):
             if isinstance(self.data_dict["unit"], list | np.ndarray)
             else [self.data_dict["unit"]]
         )
+        if isinstance(value_as_list, np.ndarray):
+            value_as_list = value_as_list.tolist()
+        if isinstance(unit_as_list, np.ndarray):
+            unit_as_list = unit_as_list.tolist()
```

Comment:
The following would be more pythonic:

```
try:
    value_as_list = value_as_list.tolist()
except AttributeError:
    pass
```

Reply:
But this would replace two lines with four lines? (for both value_as_list and unit_as_list).

Would that really be more pythonic? (I don't care, but I don't like more lines)

Reply:
The "pythonic way" is less about the number of lines but on number of operations or the old saying "easier to ask forgiveness than permission". So I would vote for try..except. I am approving anyway though.

Reply:
ok...added the try and except although I am not convinced at all. Thanks!